cirq/contrib/quimb/state_vector.py

```diff
@@ -7,7 +7,17 @@
 
 import cirq
 
-QUIMB_VERSION = tuple(int(x) for x in quimb.__version__.split('.'))
+
+# coverage: ignore
+def _get_quimb_version():
```

Comment:
Should we have a short docstring (specifically what it is returning)?

Reply:
added!